radicle-cli/src/commands/self.rs

```diff
@@ -17,7 +17,7 @@ Usage
 
 Options
 
-    --nid                Show your Node ID
+    --nid                Show your NID
```

Comment:
could also make it "Show your NID (Node ID)"

Reply:
I'd prefer `Show your Node ID (NID)`